addons/test_mail_sms/models/test_mail_sms_models.py

```diff
@@ -46,7 +46,7 @@ class MailTestSmsBl(models.Model):
     @api.depends('customer_id')
     def _compute_mobile_nbr(self):
         for phone_record in self.filtered(lambda rec: not rec.mobile_nbr and rec.customer_id):
-            phone_record.mobile_nbr = phone_record.customer_id.mobile
+            phone_record.mobile_nbr = False
```

Comment:
Would remove the compute completely, it is now a simple additional field people can fill (testing multi phone fields models). No need to "reset" when changing a field, better keep test models simple.